fleetspeak/src/client/daemonservice/execution/execution.go

```diff
@@ -91,6 +91,11 @@ type Execution struct {
 
 	inProcess   sync.WaitGroup         // count of active goroutines
 	startupData chan *fcpb.StartupData // Startup data sent by the daemon process.
+
+	heartbeat         int64 // Time of the last input message in seconds since epoch (UTC).
```

Comment:
Let's also mention that this is "atomic access only".

Reply:
Done.